crates/accelerate/src/two_qubit_decompose.rs

```diff
@@ -0,0 +1,190 @@
+// In numpy matrices real and imaginary components are adjacent:
```

Comment:
This needs the apache header, this is a good reminder that we need to update https://github.com/Qiskit/qiskit/blob/main/tools/verify_headers.py to look at rust source files too:

```suggestion
// This code is part of Qiskit.
//
// (C) Copyright IBM 2023
//
// This code is licensed under the Apache License, Version 2.0. You may
// obtain a copy of this license in the LICENSE.txt file in the root directory
// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
//
// Any modifications or derivative works of this code must retain this
// copyright notice, and modified files need to carry a notice indicating
// that they have been altered from the originals.

// In numpy matrices real and imaginary components are adjacent:
```

Reply:
`Copyright IBM 2023` this won't be accurate as soon as someone else contributes to this file. But, rules are rules.